module/organisations/helper.js

```diff
@@ -5,28 +5,29 @@
  * Description : All platform organisation related information.
  */
 
-let sunBirdService =
+let sunbirdService =
     require(ROOT_PATH + "/generics/services/sunbird");
 
 let formsHelper = require(MODULES_BASE_PATH + "/forms/helper");
 
 module.exports = class OrganisationsHelper {
 
     /**
-   * Get platform organisations list.
-   * @method
-   * @name list
-    * @returns {json} Response consists of organisations.
-   */
+       * Get platform organisations list.
+       * @method
+       * @name list
+       * @param {String} token - user access token
+       * @param {String} userId - user id
+       * @param {String} pageSize - maximum limit 
+       * @param {String} pageNo - page number
+       * @returns {json} Response consists of organisations.
+      */
 
     static list(token, userId, pageSize, pageNo) {
         return new Promise(async (resolve, reject) => {
             try {
-
-
                 let profileData = await _getProfileData(token, userId);
                 if (profileData) {
-
                     let organisationsList = [];
 
                     let roles = profileData.result.response.roles;
```

Comment:
@rakeshSgr - Please use module helpers and not directly query models of other modules